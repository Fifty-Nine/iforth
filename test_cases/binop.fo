."1 2 + " 1 2 + . clear
."3 4 - " 3 4 - . clear
."5 6 * " 5 6 * . clear
."78 9 / " 78 9 / . clear
."1011 12 % " 1011 12 % . clear
."1 1 & " 1 1 & . clear
."1 0 & " 1 0 & . clear
."1 0 | " 1 0 | . clear
."0 0 | " 0 0 | . clear
."1 !   " 1 !   . clear
."0 !   " 0 !   . clear
."2 1 < " 2 1 < . clear
."1 2 < " 1 2 < . clear
."2 1 > " 2 1 > . clear
."1 2 > " 1 2 > . clear
."1 1 = " 1 1 = . clear
."1 0 = " 1 0 = . clear
