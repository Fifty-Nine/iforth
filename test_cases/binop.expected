1 2 + 3
3 4 - -1
5 6 * 30
78 9 / 8
1011 12 % 3
1 1 & 1
1 0 & 0
1 0 | 1
0 0 | 0
1 !   0
0 !   1
2 1 < 0
1 2 < 1
1 1 < 0
2 1 > 1
1 2 > 0
1 1 > 0
1 1 = 1
1 0 = 0
2 1 <= 0
1 2 <= 1
1 1 <= 1
2 1 >= 1
1 2 >= 0
1 1 >= 1
1 0 <> 1
1 1 <> 0
